.github/workflows/ci.yaml

```diff
@@ -14,20 +14,17 @@ jobs:
 
         runs-on: ubuntu-latest
 
+        container: python:3.7-alpine
+
         steps:
             -   name: "Checkout"
                 uses: actions/checkout@v2
 
-            -   name: "Set up Python 3.7"
-                uses: actions/setup-python@v1
-                with:
-                    python-version: '3.7' # Semantic version range syntax or exact version of a Python version
-
             -   name: "Display Python version"
                 run: python -c "import sys; print(sys.version)"
 
             -   name: "Install Sphinx dependencies"
-                run: sudo apt-get install python-dev build-essential
+                run: apk add --no-cache git make
```

Comment:
Do we need git?

Reply:
needed for the pip dependencies. Name updated.